### ISSUE TITLE
Unresolved reference: stellar

### ISSUE THREAD

**Issue by let-it-snow** [stellar/java-stellar-sdk#460]
### What version are you using?
**3.5.0** - **3.8.0** 

### What did you do?
After updating from the version **3.4.2** to the newest (3.5.0 - 3.8.0) - received error for the sdk import. This will complicate further development.

<img width="447" alt="Screen Shot 2022-12-23 at 3 05 33 PM" src="https://user-images.githubusercontent.com/45386272/209341331-ad43bbef-a6fc-4977-b62b-2643cdda4049.png">

Steps to reproduce:

 - Create a new Kotlin project in the Android Studio.
 - Apply Java Stellar SDK 0.35.0 - 0.38.0.
 - Try to create any object from the SDK.
 - See the import: unresolved reference: stellar.

### What did you expect to see?
Fix or suggestions.

### What did you see instead?
Unresolved reference.

**Comment by sreuland** [comment]
@let-it-snow , this classpath issue is seen in Android Studio IDE only? can you run the android project's gradle build from command line - `./gradlew build`, any compile errors seen during that?  

I see similar red highlighting on the package name and warning in Studio file editor, but oddly, I can still launch the debug with emulator, and it builds project w/o compile error and able to step through the code even.

and build.gradle has been updated to ref the java sdk, like:
`implementation 'com.github.stellar:java-stellar-sdk:0.37.2'`

**Comment by let-it-snow** [comment]
>  this classpath issue is seen in Android Studio IDE only? 

Yes. Looks like it hapenned only on the latest stable Android Studio Electric Eel | 2022.1.1 Patch 1 and before. Checked also on the latest IntelliJ IDEA and Android Studio Flamingo - it works fine. Maybe gradle version or something else made impact.

**Comment by sreuland** [comment]
@let-it-snow , is this effectively obsolete, will close if so, please advise, thanks!

**Comment by let-it-snow** [comment]
@sreuland, can be closed for now. Thanks.
